lib/ios.js

```diff
@@ -0,0 +1,127 @@
+import {cp, fs, macOsxVersion} from './utils';
```

Comment:
ultimately I think this whole file should become part of the appium-ios package and just imported here